database/factories/RockTheVoteLogFactory.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Models\RockTheVoteLog;
+use App\Models\User;
+use Carbon\Carbon;
+use Faker\Generator as Faker;
+
+$factory->define(RockTheVoteLog::class, function (Faker $faker) {
+    return [
+        'finish_with_state' => 'No',
+        'import_file_id' => $faker->randomDigitNotNull,
+        'pre_registered' => 'No',
+        'started_registration' => Carbon::now()->format('Y-m-d H:i:s O'),
+        'status' => 'Step 1',
+        'tracking_source' => 'ads',
+        'user_id' => factory(User::class)->create(),
```

Comment:
This needs an `->id` at the end I think! And might be good to place in a closure in case we override:

```suggestion
        'user_id' => function () {
            return factory(User::class)->create()->id;
        },
```

Reply:
Whoops, I forgot we were trying out auto-merge! 😂

Reply:
We'll probably want to apply the same fix [here](https://github.com/DoSomething/northstar/blob/27740d4e9db172bb5bbd248badd9c35a2da4957e/database/factories/PostFactory.php#L33) as well!